foundation/src/main/java/com/decathlon/vitamin/compose/vitaminicons/line/ChartLine.kt

```diff
@@ -0,0 +1,48 @@
+package com.decathlon.vitamin.compose.vitaminicons.line
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType.Companion.NonZero
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap.Companion.Butt
+import androidx.compose.ui.graphics.StrokeJoin.Companion.Miter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.ImageVector.Builder
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.unit.dp
+import com.decathlon.vitamin.compose.vitaminicons.LineGroup
+
+public val LineGroup.ChartLine: ImageVector
```

Comment:
This vector has 2 different names in Fill or Line 
Line.ChartLine 
vs
Fill.LineChart

Reply:
As discussed together @ManonPolle, it's probably a problem of parsing inside the script because of the word "line".
Here, "line" doesn't mean "line" vs. "fill" but just explains "line-chart", as other icons for dashboarding like "bar-chart" or "pie-chart" :-)